client/src/providers/transactions/create.tsx

```diff
@@ -31,53 +31,56 @@ export function createTransaction(
   const accountIndex = trackingId % feeAccounts.length;
   const programDataAccount = programAccounts[accountIndex];
   const feeAccount = feeAccounts[accountIndex];
-  const instruction = new TransactionInstruction({
-    keys: [{ pubkey: programDataAccount, isWritable: true, isSigner: false }],
-    programId,
-    data: Buffer.from(Bytes.instructionDataFromId(bitId)),
-  });
 
-  const transaction = new Transaction();
-  transaction.add(instruction);
+  workerRPC
+    .createTransaction({
+      trackingId: trackingId,
+      blockhash: blockhash,
+      programId: programId.toBase58(),
+      programDataAccount: programDataAccount.toBase58(),
+      bitId: bitId,
+      feeAccountSecretKey: feeAccount.secretKey,
+    })
+    .then(
+      (response: CreateTransactionResponseMessage) => {
+        const { signature, serializedTransaction } = response;
 
-  const sentAt = performance.now();
-  transaction.recentBlockhash = blockhash;
-  transaction.sign(feeAccount);
-  const signatureBuffer = transaction.signature;
-  if (!signatureBuffer) throw new Error("Failed to sign transaction");
-  const signature = bs58.encode(signatureBuffer);
-  const pendingTransaction: PendingTransaction = { sentAt };
-  pendingTransaction.timeoutId = window.setTimeout(() => {
-    dispatch({ type: ActionType.TimeoutTransaction, trackingId });
-  }, SEND_TIMEOUT_MS);
+        const sentAt = performance.now();
```

Comment:
As mentioned over DM, the transaction is currently not sent right away. It will only be sent if retries are enabled and so will, at best, be delayed 500ms otherwise not sent at all

Reply:
@oJshua do you think you could get to this today?

Reply:
@jstarry Yes I can

Reply:
A little bit confused as to where that setTimeout() containing the pending transaction retry interval came from, but it seemed like it was in the original code for a reason. It doesn't appear to be in the current master, so I axed it.